src/pages/items/ItemCard.js

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import heartIcon from "../../asset/icon/ic_heart.png";
+import defaultImage from "../../asset/none.jpeg";
+
+function ItemCard({ item, imgSizeClass }) {
+  const imageUrl =
+    item.images && item.images.length > 0 ? item.images[0] : defaultImage;
```

Comment:
item 이 없을 경우에도 대비해주면 좋을 거 같아요!
```suggestion
    item?.images && item.images.length > 0 ? item.images[0] : defaultImage;
```